src/PHaaS/GroupClient.php

```diff
@@ -26,4 +27,90 @@ public function getAll($page = 1, $perPage = 15, $filters = [])
 
         return $groups;
     }
+
+    /**
+     * Create a group
+     *
+     * @param $name
+     * @param array $userIds
+     * @return Group
+     */
+    public function createGroup($name, $userIds = [])
+    {
+        $payload = json_encode([
+            'name'     => $name,
+            'user_ids' => $userIds
+        ]);
+
+        $response = $this->request(
+            'POST',
+            'v1/groups',
+            $payload,
+            ['Content-Type' => 'application/json']
```

Comment:
Seems a bit strange to manually set the content type to json, is this necessary?